pre-commit.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+function red() {
```

Comment:
The `function` is bash syntax, not sh.
```suggestion
red() {
```